fastlane/docs/Plugins.md

```diff
@@ -0,0 +1,64 @@
+# fastlane Plugins
+
+## Local actions
+
+You can create your own actions to extend the functionality of fastlane. The action you create will behave exactly like the built in actions.
+
+Just run `fastlane new_action`, enter the name of the action and edit the generated Ruby file in `fastlane/actions/[action_name].rb`. After you finished writing your action, add it to your version control, so it is available for your whole team.
+
+From then on, you can just use your action in your `Fastfile`, just like any other action.
+
+### Submitting the action to the fastlane main repo
+If you think your plugin will be useful to other developers as well, clone fastlane, copy your local action to fastlane/lib/fastlane/actions and submit a pull request.
+**TODO**: Add more information about what kind of PRs we want to merge
```

Comment:
Okay based on my comment for this todo what about this text:

```
Please be aware we may not accept all actions submitted to be included with fastlane.

This so we can focus on those that are:

- Universal: Bundled actions should work for tools that are typically installed on a developers machine such as Xcode or Android Studio rather than unique or edge case setups.

We reccomend custom actions or plugin actions for any unique setup you may have.

- Commonly Needed: Bundled actions should fulfill a common need such as generating certificates, bumping build versions or building the app.

We reccomend using custom actions or writing the steps In the fastfile for any one-off or unique actions you wish to achieve.
```